README.md

```diff
@@ -145,15 +176,17 @@ Retrieve the current setting for repeat
     - `NO_REPEAT` will not repeat the queue
     - We created constants named `GMusic.Playback.LIST_REPEAT`, `GMusic.Playback.SINGLE_REPEAT`, `GMusic.Playback.NO_REPEAT`
 
-#### `playback.toggleRepeat(mode)`
+#### `playback.setRepeat(mode)`
 Change the current setting for repeat
 
-- mode `String` - Optional mode to change repeat to
-    - If not specified, we will toggle to the next mode
-        - The order is `NO_REPEAT`, `LIST_REPEAT`, `SINGLE_REPEAT`
+- mode `String` - Required mode to change repeat to
     - Valid values are `NO_REPEAT`, `LIST_REPEAT`, `SINGLE_REPEAT`
         - See `playback.getRepeat()` for meaning
 
+#### `playback.toggleRepeat()`
+Toggle through the modes for repeat.
+- The order is `NO_REPEAT`, `LIST_REPEAT`, `SINGLE_REPEAT`
```

Comment:
For consistency with the rest of the markdown, please include a newline between the paragraph and list:

```
Toggle through the modes for repeat

- The order is `NO_REPEAT`, `LIST_REPEAT`, `SINGLE_REPEAT`
```